bin/i18n/i18n_script_utils.rb

```diff
@@ -321,10 +321,18 @@ def self.sanitize_data_and_write(data, dest_path)
   end
 
   def self.parse_file(file_path)
-    return JSON.load_file(file_path) if json_file?(file_path)
-    return YAML.load_file(file_path) if yaml_file?(file_path)
+    file_content = File.read(file_path)
+
+    return JSON.parse(file_content) if json_file?(file_path)
+    return YAML.safe_load(file_content) if yaml_file?(file_path)
 
     raise "do not know how to parse file #{file_path.inspect}"
+  rescue Errno::ENOENT => exception
+    raise "File not found: #{file_path.inspect} - #{exception.message}"
+  rescue JSON::ParserError => exception
+    raise "JSON parsing error in file #{file_path.inspect} - #{exception.message}"
+  rescue Psych::SyntaxError => exception
+    raise "YAML parsing error in file #{file_path.inspect} - #{exception.message}"
```

Comment:
These changes are completely arbitrary, I just wanted to produce more detailed Error logs, and handle `File.read` errors separately from parsing errors.